examples/upload.py

```diff
@@ -1,4 +1,3 @@
-from __future__ import print_function
 from past.builtins import xrange
 from StringIO import StringIO
```

Comment:
The `StringIO` module [is not available in Python 3](https://docs.python.org/3.5/whatsnew/3.0.html?highlight=stringio):

> The StringIO and cStringIO modules are gone. Instead, import the [io](https://docs.python.org/3.5/library/io.html#module-io) module and use [io.StringIO](https://docs.python.org/3.5/library/io.html#io.StringIO) or [io.BytesIO](https://docs.python.org/3.5/library/io.html#io.BytesIO) for text and data respectively.

```suggestion
from io import StringIO
```